deploy/kubernetes/helm/values.yaml.template

```diff
@@ -33,6 +33,15 @@ image: apache/heron:VERSION
 # Heron image pull policy
 imagePullPolicy: IfNotPresent
 
+# Optional Variables
```

Comment:
Let's move these under the `bookkeeper` section (@ line 109). Maybe insert at the end of that section (@ line 116). You will also need to update the property names in the yaml to reference `.Values.bookkeeper.*`